bokeh/util/tests/test_testing.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
```

Comment:
Hah, yah I know but I am trying to be consistent with the naming of test modules :)